File: Python/problem2.py
numbers = [1,2]
sum = 0
while numbers[len(numbers)-1] <= 4*10**6:
    n = numbers[len(numbers)-1]+numbers[len(numbers)-2]
    numbers.append(n)
    if(n % 2 == 0):
        sum += n

print(sum)

File: C#/problem2/problem2.cs
using System;

class Problem2{

  static void Main(){
    int a = 1;
    int b = 2;
    int sum = 0;
    while(b < 4000000){
      if(b % 2 == 0)
        sum += b;
      int temp = b;
      b = a + b;
      a = temp;
    }
    Console.WriteLine(sum);
  }

}
